docker-compose.yml

```diff
@@ -155,5 +155,11 @@ services:
     entrypoint: 'psql -h persistent_db -U postgres'
     depends_on: ['persistent_db']
 
+  watch:
```

Comment:
Would you mind changing this to something more specific like `ptw` to distinguish it from the Node version?

Reply:
Sure.